apps/build-onchain-apps/app/layout.tsx

```diff
@@ -0,0 +1,39 @@
+import '../src/global.css';
+
+import { Roboto_Mono, Inter } from 'next/font/google';
+import OnchainProviders from '../src/providers/OnchainProviders';
+import { initAnalytics } from '../src/utils/analytics';
+import type { Metadata } from 'next';
+
+const roboto = Roboto_Mono({
```

Comment:
This kind of hurts my eyes.... so next has some sort of cool library that makes some of this working like magic..... mmmmmmm

Reply:
no it doesn't work like magic, which I also assumed. These fonts are actually broken now, so still we need to update something so they get picked up everywhere again...